idaes/core/util/model_diagnostics.py

```diff
@@ -145,20 +145,20 @@ def check_residuals(self, tol=1e-5, print_level=2, sort=True):
             if sort:
                 residual_values = dict(sorted(residual_values.items(), key=itemgetter(1),reverse=True))
 
-            for i, (c,r) in enumerate(residual_values.items()):
-                if print_level == 1:
-                    # Basic print statement. count, constraint, residual
-                    print(i,"\t",c,"\t",r)
-                else:
-                    # Pretty print constraint
-                    print("\ncount =",i,"\t|residual| =",r)
-                    c.pprint()
-        
-                if print_level == 2:
-                    # print values and bounds for each variable in the constraint
-                    print("variable\tlower\tvalue\tupper")
-                    for v in identify_variables(c.body):
-                        self.print_variable_bounds(v)
+                for i, (c,r) in enumerate(residual_values.items()):
+                    if print_level == 1:
+                        # Basic print statement. count, constraint, residual
+                        print(i,"\t",c,"\t",r)
+                    else:
+                        # Pretty print constraint
+                        print("\ncount =",i,"\t|residual| =",r)
+                        c.pprint()
+        
+                    if print_level == 2:
+                        # print values and bounds for each variable in the constraint
+                        print("variable\tlower\tvalue\tupper")
+                        for v in identify_variables(c.body):
+                            self.print_variable_bounds(v)
             else:
                 print("No constraints with residuals larger than",tol,"!")
```

Comment:
@adowling2 you might want to double check this change. If I'm following your code I think the indentation on the for loop was correct and the problem is with the `else` statement on line 162. I think that needs to line up with the `if` on line 140. With this change nothing will be printed if `sort=False` which I don't think is what you intended.

Reply:
I'd think a unit test for this would help here?